File: tests/cases/comparison.py
def compare_with_integer_variable():
    i: int = 0
    s: int = 1
    if i:
        s = 2
    else:
        s = 3

    assert s == 3


def use_zero_for_comparison():
    i: int = 0
    s: int = 1
    if 0:
        s = 2
    else:
        s = 3

    assert s == 3


if __name__ == "__main__":
    compare_with_integer_variable()
    use_zero_for_comparison()
    print("OK")

File: tests/expected/comparison.jl
function compare_with_integer_variable()
    i::Int64 = 0
    s::Int64 = 1
    if i != 0
        s = 2
    else

        s = 3
    end
    @assert(s == 3)
end

function use_zero_for_comparison()
    i::Int64 = 0
    s::Int64 = 1
    if false
        s = 2
    else

        s = 3
    end
    @assert(s == 3)
end

function main()
    compare_with_integer_variable()
    use_zero_for_comparison()
    println(join(["OK"], " "))
end

main()
